internal/js_library/v2/babel.js

```diff
@@ -0,0 +1,45 @@
+// A simple wrapper around babel-core due to https://github.com/babel/babel/issues/8193
```

Comment:
note, now that we have "idiomatic install", the existence of this helper code suggests there should be a `@bazel/babel` package containing this JS, and its package.json should have the dependencies currently listed in `internal/js_library/v2/package.json`, and we don't need the extra `build_bazel_rules_nodejs_js_library_deps`.

(Doing it the way you've done is an option too, but we are moving away from that, can discuss tradeoffs more)

Reply:
Cool, I did see some happenings around this but was not fully aware - happy to do so.

Reply:
Done - Then I suppose I just add a `npm_package` rule and what is the best way to make it deploy?

Reply:
Also one thing that does feel a bit odd now (if I have done everything correctly of course) that now we always have to specify a custom babel if the deps are not installed under the `@npm` workspace. But I guess in most cases that should be fine and people should be able to just install it under the default.